### PR TITLE
Added SceThreadmgrForDriver nids

### PR DESCRIPTION
**Pull request by TheOfficialFloW**

### CODE REVIEW

File: db.yml [hunk 1]
```diff
@@ -1818,37 +1818,70 @@ modules:
         kernel: true
         nid: 0xE2C40624
         functions:
+          ksceKernelCancelCallback: 0xC040EC1C
           ksceKernelCancelMutex: 0x7204B846
+          ksceKernelChangeThreadPriority2: 0x63DAB420
```
Comment (devnoname120) [inline db.yml]: Why?

Comment (TheOfficialFloW) [inline db.yml]: http://pastebin.com/iGFQ9jfe

File: db.yml [hunk 2]
```diff
@@ -1818,37 +1818,70 @@ modules:
...
+          ksceKernelSignalCond: 0xAC616150
+          ksceKernelSignalCondAll: 0x6EC78CD0
+          ksceKernelSignalCondTo: 0x61533DA9
+          ksceKernelSignalSema: 0xD270498B
           ksceKernelStartThread: 0x21F5419B
+          ksceKernelStartTimer: 0x84C4CE4D
+          ksceKernelStopTimer: 0x474F214B
+          ksceKernelTryLockReadRWLock: 0xFC2B5A50
+          ksceKernelTryLockWriteRWLock: 0xA96F2E5A
+          ksceKernelUnlockMutex: 0x1E82E5D0
```
Comment (devnoname120) [inline db.yml]: Duplicated too.

File: db.yml (hunk 1 above)
Comment (devnoname120) [inline db.yml]: `sceKernelChangeThreadPriority` and `sceKernelChangeThreadPriority2` have different NIDs, while in this change they have the same one. Were those two functions merged?

Comment (TheOfficialFloW) [inline db.yml]: True. Changed

**Pull request merged**